src/main/java/p0nki/pesl/api/builtins/PESLBuiltins.java

```diff
@@ -110,6 +111,12 @@
                     value = arguments.get(i).asBoolean().getValue();
                 }
                 return new BooleanObject(value);
+            }))
+            .put("noise", FunctionObject.of(false, arguments -> {
+                PESLEvalException.validArgumentListLength(arguments, 1, 2);;
```

Comment:
Double semicolon :irritatered:

Very bad pr, so many conflicts with other prs and so many problems, trying to sneak in bad names but I hereby block your malicious attempt.